src/app/organizations/manage/events.component.ts

```diff
@@ -64,8 +68,25 @@ export class EventsComponent implements OnInit {
         this.loaded = true;
     }
 
+    async exportEvents() {
+        if (this.appApiPromiseUnfulfilled()) {
+            return;
+        }
+
+        this.loading = true;
+        try {
+            this.exportPromise = this.exportService.getEventExport(this.events);
+            const data = await this.exportPromise;
+            const fileName = this.exportService.getFileName('org-events', 'csv');
+            this.platformUtilsService.saveFile(window, data, { type: 'text/plain' }, fileName);
+        } catch { }
```

Comment:
I thought we were moving away from empty `try/catch` blocks (or at least putting a debug message in there). Is there any particular error you're expecting here that you want to swallow?

Reply:
I totally agree. I didn't because the error handling is done in [api-action.directive](https://github.com/bitwarden/jslib/blob/master/src/angular/directives/api-action.directive.ts)

Now that I look again, though I want to combine all the save stuff in there too.